lib/generators/sufia/templates/catalog_controller.rb

```diff
@@ -4,9 +4,8 @@ class CatalogController < ApplicationController
   include Sufia::Catalog
   include BlacklightAdvancedSearch::Controller
 
-  # These before_filters apply the hydra access controls
-  before_filter :enforce_show_permissions, only: :show
-  skip_before_filter :default_html_head
```

Comment:
`default_html_head` doesn't exist, and Rails 5 gives you an error if you try to skip something that doesn't exist.